packages/swapper/src/api.ts

```diff
@@ -64,6 +64,7 @@ export type SupportedSellAssetsInput = {
 }
 
 type CommonTradeInput = {
+  bip44Params?: BIP44Params
```

Comment:
should this be required?

Reply:
It becomes required on the stacked PR (https://github.com/shapeshift/lib/pull/1037) - we can't make this required until we remove `sellAssetAccountNumber` or a bunch of places will be sad